src/backend/marsha/core/models/video.py

```diff
@@ -509,6 +509,13 @@ class Meta:
         db_table = "video_thumbnail"
         verbose_name = _("thumbnail")
         ordering = ["-created_on", "id"]
+        constraints = [
+            models.UniqueConstraint(
+                fields=["video"],
+                condition=models.Q(deleted=None),
+                name="thumbnail_video_unique_idx",
+            )
+        ]
```

Comment:
Maybe we should add a comment (taken from the commit message) to explain why we need this ?

Reply:
Done

Reply:
:+1: